packages/registry/src/index.ts

```diff
@@ -181,6 +182,84 @@ class Config {
       }
     }
   }
+
+  getSlot<T>(name: string, args: T): SlotComponent['component'][] | undefined {
+    const { slots, data } = this._data.slots[name];
+    const slotComponents = [];
+    // For all enabled slots
+    for (const slotName of slots) {
+      // For all registered components for that slot, inversed, since the last one registered wins
+      // TODO: Cover ZCA use case, where if more predicates, more specificity wins if all true.
+      // Let's keep it simple here and stick to the registered order.
+      for (const slotComponent of data[slotName].toReversed()) {
```

Comment:
You can use `find()` here, similar to https://github.com/plone/volto/blob/e7788b3358e56bc0cc13a650d56915ed6984e396/packages/volto-slate/src/blocks/Text/keyboard/index.js#L30